e2e/transform/transform-environment/environment.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Config} from '@jest/types';
+import NodeEnvironment from 'jest-environment-node';
+
+export default class CustomEnvironment extends NodeEnvironment {
+  constructor(config: Config.ProjectConfig) {
+    super(config);
+    this.global.one = 1;
```

Comment:
I observed that the typescript compiler seems to report setting `this.global.one` as an error, because of the type `Global`

Reply:
Doesn't it allow you to add extra things at all? Seems sub optimal...

Reply:
Maybe update the type into something like this 

https://www.typescriptlang.org/play/index.html#code/JYOwLgpgTgZghgYwgAgCoQM5mQbwFDLID2IEAXMiAK4C2ARtADQHIDaEAHmFHBhVlFABzALoU4IAJ54AvnjwAbCNkhZkFdGoC8uEuWQBGGQG55qsADowAdyLIdAJlNA

Reply:
Makes sense. Wanna send a separate PR?

Reply:
Would be happy to

Reply:
Raised a PR for this